web/rest/user/src/Controller/My/CallHistoryAction.php

```diff
@@ -73,10 +73,6 @@ protected function setUserTimezone(UserInterface $user, \Traversable $calls): vo
             $call
                 ->getStartTime()
                 ->setTimezone($timezone);
-
-            $call
```

Comment:
why?

Reply:
We don't have endTime property in the UsersCdr Model